manifests/file.pp

```diff
@@ -38,11 +38,11 @@
     mode    => '0640',
   }
 
-  exec { "apply-${patch_name}.patch":
+  exec { "apply-${patch_name}${real_prefix}.patch":
```

Comment:
(y)

What about swapping `patch_name` and `real_prefix` so prefix comes first and the name after that? Then the order would be consistent with the `$patch_file` and prefix would be a prefix instead of a suffix.